modules/Attendance/report_graph_byType.php

```diff
@@ -80,8 +80,8 @@ function getDateRange($guid, $connection2, $first, $last, $step = '+1 day', $out
     $rollGroups = !empty($_POST['gibbonRollGroupID'])? $_POST['gibbonRollGroupID'] : array('all');
     if (in_array('all', $rollGroups)) $rollGroups = array('all');
 
-    require_once $_SESSION[$guid]['absolutePath'].'/modules/Attendance/src/attendanceView.php';
-    $attendance = new Module\Attendance\attendanceView($gibbon, $pdo);
+    require_once __DIR__ . '/src/AttendanceView.php';
+    $attendance = new AttendanceView($gibbon, $pdo);
```

Comment:
Needs a `namespace Gibbon\Module\Attendance;`

Reply:
Will add `use` statement on top for this.